index.html

```diff
@@ -44,14 +49,16 @@ <h5>LET'S CONNECT</h5>
           </ul>     
       </section>  
    </section>
-
+  </main>
+  
       <!--Work Section -1  -->
    
-      <section class="work-section">
+  <section class="work-section">
```

Comment:
Please try to follow the given Figma design. Though your design is not bad, I think it is better to follow the given desktop work-section design. 

Your Desktop version doesn't look like the one in the design you chose between the sizes 992px and 1100px. Please try adding the corresponding CSS property to stick to the design as much as possible (e.g. margins, padding, etc), 

Check the images below to have a better understanding. 
**Figma design:** 
![Capture2](https://user-images.githubusercontent.com/75709904/144512804-3b8518ba-ac39-45d8-994b-890f88ff99cf.PNG)
**Your design:** 
![Capture](https://user-images.githubusercontent.com/75709904/144512850-b26b7d8a-50fd-4603-b9c9-22a166fb15ed.PNG)